RadarSDK/Radar.m

```diff
@@ -1052,7 +1052,7 @@ + (void) logTermination {
 
 + (void) logEnteringBackground {
     [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelInfo type:RadarLogTypeNone message:@"App entering background" includeDate:YES includeBattery:YES append:YES];
-    [[RadarLogBuffer sharedInstance] flushToPersistentStorage ];
+    [[RadarLogBuffer sharedInstance] persist ];
```

Comment:
I'd go with `persistLogs` — `persist` could be easier to misinterpret / get confused by